src/reanimated2/NativeReanimated/NativeReanimated.ts

```diff
@@ -172,9 +181,21 @@ See https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooti
   getViewProp<T>(
     viewTag: number,
     propName: string,
+    component: React.Component | undefined, // required on Fabric
     callback?: (result: T) => void
   ) {
-    return this.InnerNativeModule.getViewProp(viewTag, propName, callback);
+    let shadowNodeWrapper;
+    if (isFabric()) {
```

Comment:
Not a huge fan of calling `isFabric` here, let's have two separate functions, one for Paper, one for Fabric.

Reply:
That would be a big change, since we have a lot of usages of `getViewProp` in our files, renaming it into `getViewPropPaper` and `getViewPropFabric` would require changing all these files. However I've prepared `getViewProp` for slitting in the future:
* Separated cpp code of `getViewProp` into separate functions
* changed declaration of `getViewProp` to accept either viewTag or shadowNodeWrapper